extensions/compose/src/extension.ts

```diff
@@ -192,8 +192,6 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     },
   };
 
-  providerOptions.emptyConnectionMarkdownDescription = composeDescription;
-
```

Comment:
AFAIK it should not be removed ?
it's in 1.5.x branch as well
https://github.com/containers/podman-desktop/blob/1.5.x/extensions/compose/src/extension.ts#L188

Reply:
🤦🏻‍♀️ omg great catch, was an overzealous cut when i was fixing the merge conflict, fixing